modules/knative-serving-concurrent-autoscaling-requests.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * serverless/configuring-knative-serving-autoscaling.adoc
+
+[id="knative-serving-concurrent-autoscaling-requests_{context}"]
+= Configuring concurrent requests for Knative Serving autoscaling
+
+You can specify the number of concurrent requests that should be handled by each instance of an application (revision container) by adding the `target` annotation or the `containerConcurrency` field in the revision template.
+
+Here is an example of `target` and `containerConcurrency` being used in a revision template.
+
+[source,yam]
+----
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: myapp
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/target: 50
+    spec:
+      containerConcurrency: 2
```

Comment:
I'd be interested what would be the outcome when we set `containerConcurrency` and autoscaling.knative.dev/target to different values? Does it make sense? Which one would get a preference? @markusthoemmes do you know?

Reply:
Good question @mgencur - would be good to confirm.

Reply:
If we set them to different values we infer the "most sensible". I.e.:

`target=50,cc=2` (as in the example) will cause target to be tuned down to 2 as well, because it makes no sense to target more releases than you actually want to handle

`target=50,cc=100` does exactly that. It targets 50 requests but limits hard at 100 requests. That's a sensible configuration.

Might make sense to change the example to the second example I gave here.